src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1085,6 +1093,14 @@ public WrapModeType WrapModeV
             }
         }
 
+        /// <summary>
+        /// Gets or sets the mode to adjust view size to preserve the aspect ratio of the image resource.
+        /// If this is set to be true, then the width or height, which is not set by user explicitly, can be adjusted to preserve the aspect ratio of the image resource.
+        /// AdjustViewSize works only if ImageView is added to a View having Layout.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AdjustViewSize { get; set; } = false;
+
```

Comment:
This patch doesn't seem to include a way to handle the non-layout case when AdjustViewSize is true.
I'm currently ignoring the case of removing the layout again, but it's better to keep the behavior if the layout is set during working.

Please set resize policy to dimension dependency in this setter.
(Maybe we need to check details about several cases.)

Reply:
I totally agree with you and I also want to do that.
However, if AdJustViewSize sets DimensionDependency automatically, then it may misunderstand user's needs.

case 1. User sets AdjustViewSize true, and then user sets View's SizeWidth or SizeHeight later.
case 2. User sets AdjustViewSize true, and then user sets View's WidthResizePolicy or HeightResizePolicy later.

For the both cases, ImageView does not know which should be DimensionDependency when AdjustViewSize is set.

Reply:
I read the reply of the original issue page after I've written this comment................ Sorry.
Yes, I agree with that, beside the cases you said, we need to consider several complex cases like to reset a layout of a View to null. (You already mentioned before.)
I still think to use specification and resize policy together is not good way. Let's think about the solution.
For the purpose, this patch is acceptable If we adds new patch for them soon.